tests/e2e/scenarios/iperf3-attached/resources/iperf3-hub/attached-connector-binding.yml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: skupper.io/v2alpha1
+kind: AttachedConnectorBinding
+metadata:
+  name: iperf3
+spec:
+  connectorNamespace: 4uhmh-iperf3-workload
```

Comment:
I believe the intention was to use the PREFIX variable here...

Reply:
This file is rewrite using a template that includes the prefix at each run. I can remove the file. It will be recreated every time.

Reply:
Fixed by: b2d9116d